### ISSUE TITLE
targets.dat should really be a YAML file.

### ISSUE THREAD

**Issue by weaverba137** [desihub/desidatamodel#30]
The data file data/targets/targets.dat should really be a YAML file.  As far as I can tell it is already perfectly valid YAML.

This is especially important because the format of targets.dat is significantly different from all other *.dat files in that directory.

**Comment by sbailey** [comment]
I agree with changing targets.dat -> targets.yaml and ensuring that it remains a valid yaml file in the future.  Reassigning to @moustakas since he is the most active user of this file.  Any objections, John?

**Comment by weaverba137** [comment]
PS, while looking at this file, I noticed comments such as: `BGS numbers from DESI-1377v1 (which are probably out of date)`, and `PLACEHOLDER MWS numbers; not documented or defined anywhere yet`.  So maybe @moustakas ought to check the numbers in there as well.

**Comment by moustakas** [comment]
The last time I tackled cleaning up this file I broke `desisim.targets` and ended up reverting all my changes because I didn't want to deal with the necessary code changes.  Is `desisim.targets` part of your big refactor, @sbailey?  If not then I'll put this on my list.

But note that there was some discussion with the MWS WG about the MWS target densities and the response was basically that there's no such thing as "typical values"...  The BGS target densities also don't seem to have converged--
